Makefile

```diff
@@ -15,7 +15,7 @@ CFLAGS_ETHASH := -Ideps/ethash/include -Ideps/ethash/lib/ethash -Ideps/ethash/li
 CFLAGS_CRYPTO_ALGORITHMS := -Ideps/crypto-algorithms
 CFLAGS_MBEDTLS := -Ideps/mbedtls/include
 CFLAGS_EVMONE := -Ideps/evmone/lib/evmone -Ideps/evmone/include -Ideps/evmone/evmc/include
-CFLAGS_SMT := -Ideps/ckb-smt/c
+CFLAGS_SMT := -Ideps/godwoken-scripts/c/deps/sparse-merkle-tree/c
```

Comment:
Since we can use ctx->internal_store_raw, IMO we do not need to access smt_ functions in polyjuice anymore?

Reply:
Since `common.h` depend on it, we still need to include this.
```
deps/godwoken-scripts/c/common.h:6:10: fatal error: ckb_smt.h: No such file or directory
 #include "ckb_smt.h"
          ^~~~~~~~~~~
```